Add Dockerfile and make npm start Work

Methrat0n opened the pull request:
I've make a simple dockerfile to help, will be simple to change if needed.
Also, I've add the dependencies in the package.json, and add a npm start command.

Methrat0n committed:
Change the package.json so that 'npm install && npm start' will make the project work. Also add a dockerfile

Methrat0n committed:
correction of a little forgotten bug

Methrat0n committed:
Add docker explanation to the README

pigne commented:
Could you add a comment in the Readme to illustrate your change for running the script (`npm run start`) ? will it work with the parameters (e.g. `--doker=XXX`) ?

Methrat0n committed:
Update README.md

Change the parameters so that it will now work. The problem was babel handling them differently.

Methrat0n commented:
I just read that it should work without any change. I will change it again so that you can merge again. Sorry.

pigne commented:
Also you should state that one need to execute the `npm run start` command in order to run the thing.
